src/js/app.js

```diff
@@ -1,3 +1,10 @@
+/* global Materialize */
+/* global FileReader */
+/* global angular */
+/* global InvertedIndexClass */
+/* global document */
+/* global $ */
```

Comment:
The globals field configure global variable and make class, variable and libraries available globally.